hackathon/02_run_pipelines.ipynb

```diff
@@ -0,0 +1,215 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "JAPoU8Sm5E6e"
+   },
+   "source": [
+    "# MLOps Hackathon\n",
+    "\n",
+    "This hackathon is based on the [open-source Turbo Template](https://github.com/teamdatatonic/vertex-pipelines-end-to-end-samples).\n",
+    "Through this notebook series you'll get hands-on with the template and Google Cloud.\n",
+    "The hackathon is structured into the following exercises:\n",
+    "\n",
+    "1. [Health check](./01_health_check.ipynb)\n",
+    "1. **[Run pipelines](./02_run_pipelines.ipynb) - this notebook**\n",
+    "1. [Promote model](./03_promote_model.ipynb)\n",
+    "1. [Challenge: Model monitoring](./04_monitoring_challenge.ipynb)\n",
+    "1. [Challenge: Real-time predictions](./05_realtime_challenge.ipynb)\n",
+    "\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "In this notebook you'll run your first training and prediction pipelines. \n",
+    "You'll learn about containers, pipelines, caching, and `make` as a productivity accelerator."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Authenticate\n",
+    "\n",
+    "Set your project ID and authenticate using your Google Account:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "VERTEX_PROJECT_ID = \"my-project-id\"\n",
+    "GOOGLE_ACCOUNT = \"user@company.com\"\n",
+    "! gcloud config set project {VERTEX_PROJECT_ID}\n",
+    "! gcloud config set account {GOOGLE_ACCOUNT}\n",
+    "! gcloud auth login"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%cd .."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run Training Pipeline\n",
+    "\n",
+    "Vertex AI Pipelines uses KubeFlow to orchestrate your training steps, as such you'll need to:\n",
+    "\n",
+    "1. Compile the pipeline\n",
+    "1. Build dependent Docker containers\n",
+    "1. Run the pipeline in Vertex AI\n",
+    "\n",
+    "The already templated training pipeline will execute a pipeline similar to the image below in Vertex AI:\n",
+    "\n",
+    "![Training Pipeline](../docs/images/training_pipeline.png)\n",
+    "\n",
+    "Don't worry about executing steps 1-3 manually (and each time you run your pipeline!), simply run the following command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! make training wait=true"
```

Comment:
Isn't working for me. I get this error:

make[1]: Entering directory '/home/jupyter/mlops-hackathon'
make[2]: Entering directory '/home/jupyter/mlops-hackathon'
################################################################################
# Compile training pipeline
################################################################################
'CONTAINER_IMAGE_REGISTRY'
make[2]: *** [Makefile:58: compile] Error 1
make[2]: Leaving directory '/home/jupyter/mlops-hackathon'
make[1]: *** [Makefile:85: run] Error 2
make[1]: Leaving directory '/home/jupyter/mlops-hackathon'
make: *** [Makefile:105: training] Error 2

Reply:
Ignore - forgot to update env.sh